lambdas/service/app.py

```diff
@@ -964,6 +966,15 @@ def list_catalogs():
     return app.catalog_controller.list_catalogs()
 
 
+def accessible_source_ids(catalog: CatalogName,
```

Comment:
Try to add new functionality in controllers, not `app.py`. We're in the process of breaking `app.py` up into controllers.